python/mxnet/optimizer.py

```diff
@@ -650,6 +650,47 @@ def update(self, index, weight, grad, state):
         weight[:] -= current_delta + wd * weight
 
 @register
+class Ftrl(Optimizer):
+    """
+    Reference:Ad Click Prediction: a View from the Trenches
+    """
```

Comment:
Add more explanations about the arguments similar to https://github.com/dmlc/mxnet/blob/master/python/mxnet/optimizer.py#L279-L294.